src/lib/db/index.ts

```diff
@@ -86,12 +86,12 @@ export default class Database {
     return this.db().demand().select(demandColumns).where({ id })
   }
 
-  getDemandWithAttachment = async (capacityId: UUID, subtype: DemandSubtype) => {
+  getDemandWithAttachment = async (id: UUID) => {
     return this.db()
       .demand()
       .join('attachment', 'demand.parameters_attachment_id', 'attachment.id')
       .select()
-      .where({ 'demand.id': capacityId, subtype })
```

Comment:
temporary change until we have `POST order/id/creation`, otherwise I put an order on chain in order to test match2s on chain

Reply:
it means that someone can create an `order` locally then `POST capacity/id/creation` and it still works...

Reply:
https://github.com/digicatapult/dscp-matchmaker-api/pull/45 will add it if that goes in first